intro/intro.rst

```diff
@@ -18,61 +14,57 @@ The scientist's needs
+  of classic numerical methods, plotting or data processing tools. We
+  don't want to re-program the plotting of a curve, a Fourier transform
+  or a fitting algorithm. Don't reinvent the wheel!
 
-* Easy to learn: computer science is neither our job nor our education. We
-  want to be able to draw a curve, smooth a signal, do a Fourier transform
-  in a few minutes.
+* **Easy to learn** Most scientists are not payed as programmers, neither
+  have they been trained so. They need to be able to draw a curve, smooth
+  a signal, do a Fourier transform in a few minutes.
 
-* Easy communication with collaborators, students, customers, to make the code
-  live within a lab or a company: the code should be as readable as a book.
-  Thus, the language should contain as few syntax symbols or unneeded routines
-  as possible that would divert the reader from the mathematical or scientific
-  understanding of the code.
+* **Easy communication** To keep code alive within a lab or a company:
+  the code should be as readable as a book by collaborators, students, or
+  maybe customers. Python syntax is simple, avoiding strange symbols or
+  lengthy routine specifications that would divert the reader from the
+  mathematical or scientific understanding of the code.
 
-* Efficient code that executes quickly... but needless to say that a very fast
-  code becomes useless if we spend too much time writing it. So, we need both a
-  quick development time and a quick execution time.
+* **Efficient code** Python numerical modules are computationally
+  efficient. But needless to say that a very fast code becomes useless if
+  too much time is spent writing it. Python aims for quick development
+  times and quick execution times.
 
-* A single environment/language for everything, if possible, to avoid learning
-  a new software for each new problem.
+* **Universal** Python is a language used for many different problems.
+  Learning Python avoids learning a new software for each new problem.
 
-Existing solutions
-...................
+How does Python compare to other solutions?
+............................................
 
-Which solutions do scientists use to work?
+Compiled languages: C, C++, Fortran...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**Compiled languages: C, C++, Fortran, etc.**
-
-* Advantages:
+:Pros:
 
   * Very fast. Very optimized compilers. For heavy computations, it's difficult
     to outperform these languages.
 
-  * Some very optimized scientific libraries have been written for these
-    languages. Example: BLAS (vector/matrix operations)
-
-* Drawbacks:
+:Cons:
 
   * Painful usage: no interactivity during development,
     mandatory compilation steps, verbose syntax (&, ::, }}, ; etc.),
     manual memory management (tricky in C). These are **difficult
     languages** for non computer scientists.
 
-**Scripting languages: Matlab**
+Matlab scripting language
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* Advantages:
```

Comment:
in this Matlab paragraph "comprehensive and well-organized help" -> "comprehensive help"